pkg/gui/submodules_panel.go

```diff
@@ -193,25 +193,6 @@ func (gui *Gui) forSubmodule(callback func(*models.SubmoduleConfig) error) func(
 	}
 }
 
-func (gui *Gui) handleResetRemoveSubmodule(submodule *models.SubmoduleConfig) error {
-	menuItems := []*menuItem{
-		{
-			displayString: gui.Tr.LcSubmoduleStashAndReset,
```

Comment:
This command is more working-tree specific so I'm just having that there and this keybinding will only be for removing a submodule